src/_pytest/_io/terminalwriter.py

```diff
@@ -218,14 +220,75 @@ def _highlight(
             return source
         else:
             try:
-                highlighted: str = highlight(
-                    source,
-                    Lexer(),
-                    TerminalFormatter(
+                # Establishes the style to be used.
```

Comment:
I would omit these comments, instead let's try to make the code "speak for itself" :)